packages/libraries/cli/src/base-command.ts

```diff
@@ -142,17 +145,41 @@ export default abstract class extends Command {
   }
 
   registryApi(registry: string, token: string) {
-    const client = new GraphQLClient(registry, {
-      headers: {
-        Accept: 'application/json',
-        'User-Agent': `hive-cli/${this.config.version}`,
-        Authorization: `Bearer ${token}`,
-        'graphql-client-name': 'Hive CLI',
-        'graphql-client-version': this.config.version,
-      },
-    });
+    const requestHeaders = {
+      Accept: 'application/json',
+      'User-Agent': `hive-cli/${this.config.version}`,
+      Authorization: `Bearer ${token}`,
+      'graphql-client-name': 'Hive CLI',
+      'graphql-client-version': this.config.version,
+    };
```

Comment:
Integration tests are failing with status code 415, because `content-type: application/json` header is missing here.